bigcommerce-labs/dev-docs
# Updated swagger to md generation

#### What?

Updated python template to a WIP state with headers

Changed swagger2md.js to:
- remove code sample generation until this can be refined
- change layout file until code samples need to be used
- made a change so it makes use of our templates instead of stock templates included with repo

Regenerated the files based off of these changes. 

#### How To Test

Run `node swagger2md.js` and verify no code samples are left

Run `bundle exec middleman rake` and view localhost:4567/api/v3/openapi-catalog.html to verify layout is correctly utilized
